msg/battery_status.msg

```diff
@@ -43,3 +43,12 @@ uint8 warning						# current battery warning
 
 
 uint8 MAX_INSTANCES = 4
+
+float32 average_power               # The average power of the current discharge
+float32 available_energy            # The predicted charge or energy remaining in the battery
+float32 remaining_capacity          # The compensated battery capacity remaining
+float32 full_charge_capacity        # The compensated capacity of the battery when fully charged
```

Comment:
There seems to be some overlap here with existing fields like `capacity`.

Reply:
What are the specific it can be considered to overlap? remaining_capacity, full_charge_capacity and capacity?@dagar